### ISSUE TITLE
non-static network should be set up by optimize.sh

### ISSUE THREAD

**Issue by ulab** [thatvirtualboy/horizon-linux-vm#4]
Instead of telling people to run "sudo dhclient" on each reboot, the optimize.sh script should set up this to be done automatically when not using a static configuration

**Comment by thatvirtualboy** [comment]
Theoretically that should really only need to be done once during first boot, assuming the user configures static during the script, but for sure it could be optimized!

**Comment by ulab** [comment]
Yes, but practically it needs to be done on every boot, assuming the user does not configure static during the script. ;-)

We are setting these up dynamically for a myriad of users, so we have to rely on DHCP for every created instance. Also it's not a server, so there is not really a need for a static IP.
